.github/workflows/DetectDocGeneratorChanges.yml

```diff
@@ -12,9 +12,9 @@ jobs:
 
     steps:
     - name: Configure AWS Credentials
-      uses: aws-actions/configure-aws-credentials@v1
+      uses: aws-actions/configure-aws-credentials@8c3f20df09ac63af7b3ae3d7c91f105f857d8497 #v4
       with:
-        role-to-assume: ${{ secrets.CI_AWS_ROLE_ARN }}
+        role-to-assume: ${{ secrets.CI_MAIN_TESTING_ACCOUNT_ROLE_ARN }}
```

Comment:
i think ideally we should use this account to determine which test runner to use, but i'm fine with this for now.